packages/unit-testing/stubs/crypto/certificate.js

```diff
@@ -0,0 +1,77 @@
+define(['./SignedXml', './Signer', './Verifier'], function (SignedXml, Signer, Verifier) {
```

Comment:
I think we should nest certifacte.js, SignedXml.js, Signer.js and Verifier into a the /stubs/crypto/certificate and not directly into /stubs/crypto.
My logic here is that every module has its own folder, like for instance N/auth  has its /sutbs/auth folder that contains /stubs/auth/auth.js.
Same way N/crypto/certificate module should have /stubs/crypto/certifate folder that contains all the files related to this module.

Reply:
I am not sure, the same applies to https/clientCertificate but I would rather keep the file structure as flat as possible to avoid confusion. The only case where I created subfolders is when the downloaded SuiteScript API package have them, this is to relate them easier (for instance commerce/webstore). The file structure does not change anything functionally as long as have the right module names. If you really think it should be restructured feel free to do it.